```
cmd = ['dcos', 'help', 'node'], returncode = 0
stdout = b'Display DC/OS node information\n\nUsage:\n  dcos node [flags]\n  dcos node [command]\n\nCommands:\n  decommission\n ...  --json              Print in json format\n\nUse "dcos node [command] --help" for more information about a command.\n'
stderr = b'', env = None, stdin = None

    def assert_command(
            cmd,
            returncode=0,
            stdout=b'',
            stderr=b'',
            env=None,
            stdin=None):
        """Execute CLI command and assert expected behavior.
    
        :param cmd: Program and arguments
        :type cmd: list of str
        :param returncode: Expected return code
        :type returncode: int
        :param stdout: Expected stdout
        :type stdout: bytes
        :param stderr: Expected stderr
        :type stderr: bytes
        :param env: Environment variables
        :type env: dict of str to str
        :param stdin: File to use for stdin
        :type stdin: file
        :rtype: None
        """
    
        returncode_, stdout_, stderr_ = exec_command(cmd, env, stdin)
    
        assert returncode_ == returncode, (returncode_, returncode)
>       assert stdout_ == stdout, (stdout_, stdout)
E       AssertionError
```

--- a/python/lib/dcoscli/dcoscli/data/help/node.txt
+++ b/python/lib/dcoscli/dcoscli/data/help/node.txt
@@ -1,7 +1,6 @@
 Display DC/OS node information
 
 Usage:
-  dcos node [flags]
   dcos node [command]
 
 Commands:

--- a/python/lib/dcoscli/dcoscli/data/help/service.txt
+++ b/python/lib/dcoscli/dcoscli/data/help/service.txt
@@ -1,7 +1,6 @@
 Manage DC/OS services
 
 Usage:
-  dcos service [flags]
   dcos service [command]
 
 Commands:

